specs/greeting_spec.js

```diff
@@ -1,40 +1,52 @@
 const assert = require('assert')
-const Botmock = require('botkit-mock')
 
+const { getBasicController } = require('./helpers')
 const greetingController = require('../features/greeting')
 const replies = require('../features/greeting/replies.js')
 
 describe('Sample hears controller', () => {
   beforeEach(() => {
-    this.controller = Botmock({})
-    this.bot = this.controller.spawn({ type: 'slack' })
+    this.controller = getBasicController()
     greetingController(this.controller)
   })
 
-  it(
-    'Should return any greeting if user types `hi`',
-    () => this.bot.usersInput([{
+  it('Should return any greeting if user types `hi`', async () => {
```

Comment:
without `should` 😉 
```
it 'returns '
```

http://www.betterspecs.org/
**Don't use should** section